packages/ketcher-react/src/script/ui/dialog/template/TemplateDialog.tsx

```diff
@@ -350,6 +354,7 @@ export default connect(
   (store) => ({
     ...omit(['attach'], (store as any).templates),
     initialTab: (store as any).modal?.prop?.tab || TemplateTabs.TemplateLibrary,
+    renderOptions: (store as any).editor?.render?.options,
```

Comment:
is it possible to add a shape for a store?

Reply:
Its not that easy unfortunately, kind of scared of breaking other things